eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -349,8 +352,11 @@ object NodeParams extends Logging {
       toRemoteDelay = CltvExpiryDelta(config.getInt("to-remote-delay-blocks")),
       maxToLocalDelay = CltvExpiryDelta(config.getInt("max-to-local-delay-blocks")),
       minDepthBlocks = config.getInt("mindepth-blocks"),
-      feeBase = feeBase,
-      feeProportionalMillionth = config.getInt("fee-proportional-millionths"),
+      relayParams = RelayParams(
+        getRelayFees(config.getConfig("relay.fees.public-channels")),
+        getRelayFees(config.getConfig("relay.fees.private-channels")),
+        getRelayFees(config.getConfig("relay.fees.min-trampoline")),
```

Comment:
When types are the same I like to explicitly name the arguments to prevent future footguns:

```suggestion
        publicChannelFees = getRelayFees(config.getConfig("relay.fees.public-channels")),
        privateChannelFees = getRelayFees(config.getConfig("relay.fees.private-channels")),
        minTrampolineFees = getRelayFees(config.getConfig("relay.fees.min-trampoline")),
```